test/e2e/csv_e2e_test.go

```diff
@@ -3337,7 +3337,7 @@ var _ = Describe("CSV", func() {
 		require.NoError(GinkgoT(), err)
 
 		expected := &metav1.LabelSelector{
-			MatchLabels:      map[string]string{"olm.operatorgroup/" + namespace.Name + "." + og.GetName(): ""},
+			MatchLabels:      map[string]string{fmt.Sprintf("olm.operatorgroup.uid/%s", og.GetUID()): ""},
```

Comment:
Sorry if this suggestion is overly cautious, but I'd like to see this label construction put into a method that can return an error when UID is empty. I don't think that condition can occur as this is currently written, but it's the sort of non-obvious bug that could be introduced a few months down the line. At the least, it'll reduce code duplication slightly.

Reply:
This is an e2e test - the values we expect should be hard coded should the not?

Reply:
I think making it easier to debug unexpected test failures is a valid request. I have updated the PR in response.